File: 0001 Code Chef/0001 Advitiya/implementation/0003 c#/Advitiya.cs
using System;

class Advitiya{
    static void Main(string[] args){
        string stringInput = Console.ReadLine();
        int day = Convert.ToInt32(stringInput);
        if(day == 16 || day == 17 || day == 18){
            Console.WriteLine("ADVITIYA");
        }else{
            Console.WriteLine("WAITING FOR ADVITIYA");
        }
    }
}

File: 0001 Code Chef/0001 Advitiya/implementation/0005 kotlin/advitiya.kt
fun main(args:Array<String>){
    var stringInput:String? = readLine();
    if(stringInput != null){
        var day:Int? = stringInput.toIntOrNull();
        if(day != null){
            if(day == 16 || day == 17 || day == 18){
                println("ADVITYA");
            }else{
                println("WAITING FOR ADVITIYA");
            }
        }
    }
}